rotkehlchen/tests/unit/decoders/test_eth2.py

```diff
@@ -53,9 +53,59 @@ def test_deposit(database, ethereum_inquirer, ethereum_accounts):
         ), EthDepositEvent(
             tx_hash=evmhash,
             validator_index=validator.index,
-            sequence_index=1,
+            sequence_index=2,
             timestamp=TimestampMS(1674558203000),
             balance=Balance(amount=FVal('32')),
             depositor=user_address,
         ),
     ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x3e5fd0244e13d82fC230f3Fc610bcd76b3c8217C']])
+def test_multiple_deposits(database, ethereum_inquirer, ethereum_accounts):
+    evmhash = deserialize_evm_tx_hash('0x819fe4a07894cf044f5d8c63e5c1e2294e068d05bf91d9cfc3e7ae3e60528ae5')  # noqa: E501
+    user_address = ethereum_accounts[0]
```

Comment:
it's VCRed anyway but just fyi, if you read the decoder you could have saved 3 (as many as the deposits) calls to beaconchain, by pre-adding the validators like I did here: https://github.com/rotki/rotki/blob/f49c115efa640cf8fed8266ed93c17e312fb873d/rotkehlchen/tests/integration/test_eth2.py#L26-L42

As the decoder, if you look at the code, first tries to match pubkey to validator by checking the DB